### ISSUE TITLE
[FE, BE] My VA Forms Status: (QA bug) submitted forms not showing for LOA1 users

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#93723]
## Background
On staging we should be able to see certain submitted forms (see below) when submitted by LOA1 users but we are only seeing draft forms.

**the following forms allow the user to submit as LOA1 (signed in but without being verified):**
- 21-10210 ([staging link](https://staging.va.gov/supporting-forms-for-claims/lay-witness-statement-form-21-10210/introduction))
- 21-0972 ([staging link](https://staging.va.gov/supporting-forms-for-claims/alternate-signer-form-21-0972/introduction))
- 21P-0847 ([staging link](https://staging.va.gov/supporting-forms-for-claims/substitute-claimant-form-21P-0847/introduction))

**the following  forms require the user to be LOA3 (signed in and verified) in order to submit:**
- 21-4142/21-4142A ([staging link](https://staging.va.gov/supporting-forms-for-claims/release-information-to-va-form-21-4142/introduction))
- 21-0845 ([staging link](https://staging.va.gov/supporting-forms-for-claims/third-party-authorization-form-21-0845/introduction))
- 21-0966 ([staging link](https://staging.va.gov/supporting-forms-for-claims/intent-to-file-form-21-0966/introduction))
- 20-10206 ([staging link](https://staging.va.gov/records/request-personal-records-form-20-10206/introduction))
- 20-10207 ([staging link](https://staging.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207/introduction))

Steps to reproduce:
1. Log into staging as an LOA1 user 
2. Navigate to any of the three forms linked above that an LOA1 user is able to submit 
3. Complete the steps to submit the form
4. Navigate back to My VA
5. See that the submitted form is not shown in the Benefit applications and forms section

## Tasks
- [x] Investigate the issue
- [ ] Consult with BE 
- [ ] Implement fix 

## Acceptance Criteria
- We can see forms submitted by LOA1 users on My VA on staging

**Comment by allisonlu** [comment]
After syncing with @DanielMiller-agile6 and @tpharrison, we found out:

1. when we call `/submission_statuses/` endpoint, the [controller will check](https://github.com/department-of-veterans-affairs/vets-api/blob/35cce190e129ab5256ae9c65e8650f2a3a03ca4b/app/controllers/v0/my_va/submission_statuses_controller.rb#L10) if the user is [authorized according to the Lighthouse policy](https://github.com/department-of-veterans-affairs/vets-api/blob/35cce190e129ab5256ae9c65e8650f2a3a03ca4b/app/policies/lighthouse_policy.rb#L4-L6):

```
  def access?
    user.icn.present? && user.participant_id.present?
  end
```

i.e. user needs to have both
1. a **ICN** (Integration Control Number?) and
2. **participant ID**

## Relevant Slack convos
* LOA1 accounts don't get anything from MPI nor do we attempt to call MPI to get those attributes (via [Joe Niquette](https://dsva.slack.com/archives/CBU0KDSB1/p1666368144666779))
* LOA3 accounts means ICN and EDIPI should have been established (via [Thomas Blackwell](https://dsva.slack.com/archives/C02CQP3RFFX/p1719328859931119?thread_ts=1717704172.086279&cid=C02CQP3RFFX))

## tl;dr
We _think_ LOA1 users don't have ICNs (and that's why they're not seeing submitted forms on My VA), but we can change the restriction listed in the code above. Tom will reach out to Eric to see what VFF team has done so we can stay aligned.

**Comment by ajakabcin** [comment]
Spoke with @tpharrison and @allisonlu about this and we believe we have a solution - we need to implement a more lenient check on LOA1 users that doesn't require an ICN. Tom plans to make this change on Monday to resolve this ticket.

**Comment by tpharrison** [comment]
@ajakabcin I have [this draft PR](https://github.com/department-of-veterans-affairs/vets-api/pull/18690) ready to go.  It will allow both LOA1 and LOA3 users to view their submission statuses.  Just let me know when we're set and I'll mark it as ready for review.

cc/ @allisonlu @DanielMiller-agile6

**Comment by tpharrison** [comment]
@allisonlu The PR to allow access for LOA1 users was just merged, so you should be able to test in staging on Monday.  Let me know if you have any questions.

**Comment by ajakabcin** [comment]
Confirmed on staging that I'm able to see forms I submitted (with submission in progress status) on my LOA1 account - thank you @tpharrison and @allisonlu for the quick work on this!
